Content.Server/Explosion/EntitySystems/TriggerSystem.cs

```diff
@@ -101,11 +122,24 @@ private void OnActivate(EntityUid uid, TriggerOnActivateComponent component, Act
             args.Handled = true;
         }
 
+        private void OnImplantTrigger(EntityUid uid, TriggerImplantActionComponent component, ActivateImplantEvent args)
+        {
+            Trigger(component.Owner);
+        }
+
         private void OnStepTriggered(EntityUid uid, TriggerOnStepTriggerComponent component, ref StepTriggeredEvent args)
         {
             Trigger(uid, args.Tripper);
         }
 
+        private void OnStateChanged(EntityUid uid, TriggerOnMobstateChangeComponent component, MobStateChangedEvent args)
+        {
+            if (component.MobState != args.CurrentMobState)
+                return;
```

Comment:
Should this be "<" for MobState? Like if it triggers on crit but you go straight from alive to dead it won't trigger (same with it will be triggered if you go from dead to crit from healing).

Reply:
I didn't realize you could do a comparison like this, thanks!